tools/load-testing-before-deployment.sh

```diff
@@ -39,16 +39,16 @@ run_tests() (
 
   echo ">>> Ramp up"
   echo "Do not pay attention to the results for this run."
-  molotov -w 200 -d 300 --ramp-up 60 publish_short_requests.py
+  molotov -w 200 -d 150 --ramp-up 60 publish_short_requests.py
 
   echo ">>> Run a load test operation: 200 clients during 5 minutes (1/3)"
```

Comment:
Can we also update the comments that are referencing "5 minutes" here and below?

Reply:
Oh right, I wanted to do it and forgot !! Thanks for catching it